internal/cmd/kafka/command_topic_consume_onprem.go

```diff
@@ -25,6 +25,14 @@ func (c *authenticatedTopicCommand) newConsumeCommandOnPrem() *cobra.Command {
 		Short: "Consume messages from a Kafka topic.",
 		Long:  "Consume messages from a Kafka topic. Configuration and command guide: https://docs.confluent.io/confluent-cli/current/cp-produce-consume.html.\n\nTruncated message headers will be printed if they exist.",
 		Example: examples.BuildExampleString(
+			examples.Example{
+				Text: `Consume message from topic "my_topic" providing embedded Kafka REST Proxy endpoint.`,
+				Code: "confluent kafka topic consume my_topic --url http://localhost:8090/kafka",
```

Comment:
While we're at it, do you mind changing all instances of `my_topic` to `my-topic`? We'd like to get rid of mixed dashes and underscores to make things easier to type!
```suggestion
				Text: `Consume message from topic "my-topic" providing embedded Kafka REST Proxy endpoint.`,
				Code: "confluent kafka topic consume my-topic --url http://localhost:8090/kafka",
```

Reply:
Okay got it. Will go in and update those instances 👍 Thanks @brianstrauch